src/PDE/CompFlowProblem/VorticalFlow.h

```diff
@@ -0,0 +1,308 @@
+// *****************************************************************************
+/*!
+  \file      src/PDE/CompFlowProblem/VorticalFlow.h
+  \author    J. Bakosi
```

Comment:
Aren't we removing these to support a more traditional open source development model?